back/spec/services/side_fx_user_service_spec.rb

```diff
@@ -48,4 +48,38 @@
       expect(user.follows.pluck(:followable_id)).to contain_exactly area.id
     end
   end
+
+  describe 'before_destroy' do
+    before do
+      @status_review_pending = create(:initiative_status_review_pending)
+      @status_changes_requested = create(:initiative_status_changes_requested)
+      @status_proposed = create(:initiative_status_proposed)
+      @status_threshold_reached = create(:initiative_status_threshold_reached)
+      @status_expired = create(:initiative_status_expired)
+      @status_answered = create(:initiative_status_answered)
+      @status_ineligible = create(:initiative_status_ineligible)
+    end
+
+    it "destroys the user's reactions to initiatives in a voteable status" do
+      user = create(:user)
+      initiative1 = create(:initiative, initiative_status: @status_review_pending)
+      initiative2 = create(:initiative, initiative_status: @status_changes_requested)
+      initiative3 = create(:initiative, initiative_status: @status_proposed)
+      initiative4 = create(:initiative, initiative_status: @status_threshold_reached)
+      initiative5 = create(:initiative, initiative_status: @status_expired)
+      initiative6 = create(:initiative, initiative_status: @status_answered)
+      initiative7 = create(:initiative, initiative_status: @status_ineligible)
+      _reaction1 = create(:reaction, user: user, reactable: initiative1)
```

Comment:
`review_pending_initiative` would be a bit clearer than `initiative`. Not critical though :slightly_smiling_face: